docs/core/deploying/native-aot/index.md

```diff
@@ -7,9 +7,9 @@ ms.date: 02/21/2023
 ---
 # Native AOT deployment
 
-Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps start up very quickly and use less memory. Users of the application can run it on a machine that doesn't have the .NET runtime installed.
+Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps start up quickly and use less memory. The app can run it on a machine that doesn't have the .NET runtime installed.
```

Comment:
Perhaps it's better this way:
```suggestion
Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps have faster startup time and smaller memory footprints. These apps can run on a machine without .NET runtime installed.
```

cc @jkotas

Reply:
(nit) `footprints` => `footprint`. Unless you want to make `time` plural as well.

Reply:
Update the suggestion, thanks.

Reply:
Updated the suggestion, thanks.